### PR TITLE
Build data privacy form

### LINKED ISSUE #2732: Create form in Marketo

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Create data privacy enquiry page and add new form

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/legal/data-privacy-enquiry>
- Check that the page matches the [copy doc](https://docs.google.com/document/d/1_sM2dGQNd5FRda2gITWL1jD1oWsIekAw2jCxPTPD1Ss/edit)
- Make sure the form functions as you would expect

## Issue / Card

Fixes #2732

### CODE REVIEW

File: templates/legal/data-privacy-enquiry/index.html [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+<div class="p-strip is-deep">
+  <div class="row">
+    <div class="col-8">
+      <h1>Data Protection enquiries</h1>
+      <p>Please contact us using the form below for any enquiry about how Canonical collects and processes personal data or for any other questions relating to privacy or data protection. A member of our team will be in touch with you shortly. Alternatively please contact us on <a href="mailto:dataprotection@canonical.com">dataprotection@canonical.com</a>.</p>
+
+      <form method="post" action="https://www.tfaforms.com/responses/processor" class="hintsBelow labelsLeftAligned" id="tfa_0" enctype="multipart/form-data">
+        <fieldset>
+          <h3 id="tfa_16-L">Your contact details</h3>
+          <ul>
```
Comment (Caleb-Ellis) [inline templates/legal/data-privacy-enquiry/index.html]: Could you give these a "p-list" class to remove the bullet points?

File: templates/legal/data-privacy-enquiry/index.html [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+              </label>
+              <select id="tfa_20" name="tfa_20">
+                <option value="">Please select...</option>
+                <option value="tfa_21" id="tfa_21" class="">Yes</option>
+                <option value="tfa_22" id="tfa_22" class="">No</option>
+              </select>
+            </li>
+          </ul>
+          <p>If yes, please provide their contact details:</p>
+          <ul>
```
Comment (Caleb-Ellis) [inline templates/legal/data-privacy-enquiry/index.html]: As above

File: templates/legal/data-privacy-enquiry/index.html [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+                Phone number (optional):
+              </label>
+              <input type="text" id="tfa_27" name="tfa_27">
+            </li>
+          </ul>
+        </fieldset>
+
+        <fieldset>
+          <h3>Company information</h3>
+          <ul>
```
Comment (Caleb-Ellis) [inline templates/legal/data-privacy-enquiry/index.html]: As above

File: templates/legal/data-privacy-enquiry/index.html [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+            </li>
+            <li>
+              <label for="tfa_38">Country (optional):</label>
+              <input type="text" id="tfa_38" name="tfa_38">
+            </li>
+          </ul>
+        </fieldset>
+
+        <fieldset>
+          <ul>
```
Comment (Caleb-Ellis) [inline templates/legal/data-privacy-enquiry/index.html]: As above

File: templates/legal/data-privacy-enquiry/index.html [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+                Please set out your enquiry below:
+              </label>
+              <textarea id="tfa_53" name="tfa_53" required></textarea>
+            </li>
+          </ul>
+        </fieldset>
+
+        <fieldset>
+          <p>Data collected from this survey will be handled in accordance with the Canonical <a href="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy">privacy policy</a>.</p>
+          <ul>
```
Comment (Caleb-Ellis) [inline templates/legal/data-privacy-enquiry/index.html]: As above

File: templates/legal/data-privacy-enquiry/index.html [hunk 6]
```diff
@@ -0,0 +1,215 @@
...
+            <li>
+              <label for="tfa_55">
+                If you would like to include an attachment with your request please add it here:
+              </label>
+              <input type="file" id="tfa_55" name="tfa_55">
+            </li>
+            <li>
+              <button type="submit" id="submit_button">Submit</button>
```
Comment (Caleb-Ellis) [inline templates/legal/data-privacy-enquiry/index.html]: "p-button--positive" otherwise it's a weird colour on hover

**Comment by frankiegranato** [comment]
Demo links to a 404. Never mind, I'll review this with the rest of the feature branch. I'll remove the UX Needed label.

**Comment by anthonydillon** [comment]
@steverydz I think you need to rebase on master to resolve the CI issue.

**Comment by anthonydillon** [comment]
@steverydz could you do me a favour. On the `thank-you` page could you move the `h1` inside the `col-8` please.

File: templates/legal/data-privacy-enquiry/index.html (hunk 6 above)
Comment (anthonydillon) [reply templates/legal/data-privacy-enquiry/index.html]: Could you add this class @steverydz

**Comment by steverydz** [comment]
@anthonydillon Done, done and done :)

**Pull request merged**